store/SPEC_STANDARD.md

```diff
@@ -0,0 +1,116 @@
+## What is a SDK standard?
+
+A SDK standard is a design document describing a particular protocol, standard, or feature expected to be used by the Cosmos SDK. A SDK standard should list the desired properties of the standard, explain the design rationale, and provide a concise but comprehensive technical specification. The primary author is responsible for pushing the proposal through the standardization process, soliciting input and support from the community, and communicating with relevant stakeholders to ensure (social) consensus.
```

Comment:
This seems to me to be too broad. Do you mean what is an SDK Storage or State standard?

Reply:
What do you mean by State standard?

> This seems to me to be too broad.

This is intended. The first version was specific for store features, but reviewers requested to generalize it.

Reply:
Then why is it under the store package?

Reply:
If it's not a specification standard template specifically for storage/state, then it doesn't belong in `store/`. The `docs/spec` directory would be a perfectly adequate place to put it :)